### DISCUSSION TITLE
Where are the Admin configurations stored?

### DISCUSSION THREAD

**Discussion by Gusterson** [strapi/strapi#7980] [Legacy Topics]
When I have an item, say `post` and create different fields, I can then re-order them using the drag-and-drop GUI. Nice. However when copying a project over, it's tedious to re-order all of these fields.

I copied all the files in the `api` directory, and I thought I copied all the relevant database tables as well, but the ordering of these fields did not transfer.

Where exactly is Strapi looking to determine that I want 'Title' first, followed by 'Excerpt' then 'Content' and not another order?

**Comment by Gusterson** [comment]
Looks like it's kind of in the `api/type/models/type.settings.json` file, but also kind of in a `core_store` row in the database under `plugin_content_manager_configuration_content_types::application::type.type`... 

Which takes precedence?

Following some other trouble-shooting, I deleted my `core_store` table (which fixed an immediate problem) but that also blitzed my 'placeholder' and 'description' settings. <--- Seems like **these should be user accessible and editable** (in a file) and not hidden away in an obscure DB entry...

**Reply by derrickmehaffy** [reply]
There are two places (as you noticed):

Core settings (Fields, name, ect) used to structure the model database table and are **not changeable in production**
These are stored in the model file and can be committed into source control

Visual settings (layout, alt names, descriptions, placeholders, ect) are stored in the database allowing them to be customized in production don't affect the way the model is structured at the API level, but are only visual changes.
These are stored in the database

To dive deeper into database vs file see this FAQ: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#can-i-store-my-content-manager-layout-configurations-in-the-model-settings

**Reply by Gusterson** [reply]
Excellent! Thanks. Searched all over and never came up with that FAQ entry.

**Reply by derrickmehaffy** [reply]
It's a bit hidden away, we are currently reviewing our entire documentation (and just overall resources) and will making massive overhauls to all of our documentation and resources over the next few months ;)
